### PR TITLE
documentation: ZENKO-1723_monitoring_SOFS_ingestion

### PR DESCRIPTION
**Pull request by wabernat**
This PR adds a section on monitoring SOFS OOB ingestion.

It addresses #ZENKO-1723

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernatScality** [comment]
@bert-e  bypass_build_status

File: docs/docsource/Operation-Architecture/Zenko_CLI/Monitoring_NFS-SOFS_Ingestion.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+   $ kubectl logs -f {{location}}-cosmos-rclone-{{hash}}
+
+Hence:
+
+.. code::
+
+   $ kubectl logs -f my-location-name-cosmos-rclone-84988dc9d4-9dwjl
+
+yields:
```
Comment (tmacro) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Monitoring_NFS-SOFS_Ingestion.rst:34]: `yields` -> `Yields`?

**Review by tmacro — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

File: docs/docsource/Operation-Architecture/Zenko_CLI/Monitoring_NFS-SOFS_Ingestion.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Monitoring_NFS-SOFS_Ingestion.rst:34]: Nah. It's a continuation of the sentence, not the start of a new one.

**Comment by ssalaues** [comment]
@bert-e bypass_build_status

**Pull request merged**